contrib/coreos/user-data.example

```diff
@@ -113,9 +148,13 @@ write_files:
       }
   - path: /etc/systemd/system/docker.service.d/50-insecure-registry.conf
     content: |
-        [Service]
-        EnvironmentFile=/etc/environment_proxy
-        Environment="DOCKER_OPTS=--insecure-registry 10.0.0.0/8 --insecure-registry 172.16.0.0/12 --insecure-registry 192.168.0.0/16 --insecure-registry 100.64.0.0/10"
+      [Unit]
+      Requires=flanneld.service
+      After=flanneld.service
+
+      [Service]
+      EnvironmentFile=/etc/environment_proxy
+      Environment="DOCKER_OPTS=--insecure-registry 10.0.0.0/8 --insecure-registry 172.16.0.0/12 --insecure-registry 192.168.0.0/16 --insecure-registry 100.64.0.0/10 --iptables=false"
   - path: /run/deis/bin/get_image
```

Comment:
@aledbf should the docker opts contain -ip-masq=false as we already started flannel with --ip-masq=true